django/db/models/query.py

```diff
@@ -2387,6 +2409,9 @@ def prefetch_related_objects(model_instances, *related_lookups):
                 obj_list = new_obj_list
 
 
+aprefetch_related_objects = sync_to_async(prefetch_related_objects)
```

Comment:
That's good, creating `sync_to_async` objects is not cheap. We started using this pattern so often that I wonder if we'll eventually want to get to a point where these objects are lazily created at the first call time like we did with `lazy_re`

I'd take the time to document this function as well since [the _sync_ version is documented](https://docs.djangoproject.com/en/4.2/ref/models/querysets/#django.db.models.prefetch_related_objects)

Reply:
That's interesting. Django has a pretty long initialization time. I would hate to continue to add to it.

I'm pushing that documentation change right now.